src/Redux/actions.tsx

```diff
@@ -297,18 +294,9 @@ export const updateMinQuantity = (pathParams: object, params: object) => {
   return fireRequest("updateMinQuantity", [], pathParams, params);
 };
 export const getInventorySummary = (facilityId: number, params: object) => {
-  return fireRequest(
-    "getInventorySummary",
-    [facilityId, "inventorysummary"],
-    params
-  );
-};
-export const flagInventoryItem = (params: object) => {
-  return fireRequest("flagInventoryItem", [], {}, params);
-};
-
-export const deleteLastInventoryLog = (params: object) => {
-  return fireRequest("deleteLastInventoryLog", [], {}, params);
+  return fireRequest("getInventorySummary", [], params, {
+    facility_external_id: facilityId,
+  });
```

Comment:
@sriharsh05 since this covers the majority of the Inventory form, could you also migrate for "AddInventoryForm.tsx"?

That way we could remove reference to this one too and @nihal467 can test the inventory module as a whole in one go.

Reply:
You should be able to delete references to these in this PR:

<img width="733" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/9f7fde9e-e844-42d2-bf95-8f0b76ec72f4">